app/emoji/client/emojiPicker.html

```diff
@@ -44,6 +44,9 @@ <h4 class="emoji-list-category" id="emoji-list-category-{{category.key}}">{{_ ca
 				{{/each}}
 			{{/if}}
 		</div>
-		<div class="emoji-footer">{{{_"Emoji_provided_by_JoyPixels"}}}</div>
+		<div class="emoji-footer">
+			{{#if hasPermission 'manage-emoji'}}<a id="add-custom">{{{_"Add_custom_emoji"}}}</a><br/>{{/if}}
```

Comment:
please provide a better way to break the line,  like a css class,  and insert the path  on the <a> in case the user wants to open in a new window..